<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>impiegato_search_prod_dialog</class>
 <widget class="QDialog" name="impiegato_search_prod_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>244</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>409</width>
    <height>244</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ricerca prodotto</string>
  </property>
  <widget class="QLabel" name="lbl_title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ricerca prodotto</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_search_by">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>55</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ricerca:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_search_by_name">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>per nome</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_search_by_id">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>per ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_search_query">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>123</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Query di ricerca:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_search_query">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Annulla</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_search">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ricerca</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
